src/anytag.rs

```diff
@@ -1,10 +1,14 @@
 use crate::*;
+use id3::Timestamp;
 
 #[derive(Default)]
 pub struct AnyTag<'a> {
     pub config: Config,
     pub title: Option<&'a str>,
     pub artists: Option<Vec<&'a str>>,
+    pub date_released: Option<Timestamp>,
+    pub original_date_released: Option<Timestamp>,
+    pub date_recorded: Option<Timestamp>,
```

Comment:
```suggestion
    pub date: Option<Timestamp>,
```

ID3v2 is the only tag with a concept of original dates and release times. When going generic, there should only be recording date, as that is the only date supported by all formats.

Reply:
I went ahead and generified it to `date` in AnyTag [here](https://github.com/TianyiShi2001/audiotags/pull/34/commits/6f5c5d03cb48c7bff884a4bb14382f57140a13f6), as well as AudioTagEdit [here](https://github.com/TianyiShi2001/audiotags/pull/34/commits/0042cde57d26c4f9a559471abcffc5791edf3077)